src/RelationParser.h

```diff
@@ -0,0 +1,74 @@
+//
+//  RelationParser.h
+//  snowcrash
+//
+//  Created by Pavan Kumar Sunkara on 04/03/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+
+#ifndef SNOWCRASH_RELATIONPARSER_H
+#define SNOWCRASH_RELATIONPARSER_H
+
+#include "SectionParser.h"
+#include "RegexMatch.h"
+
+namespace snowcrash {
+
+    /** Link Relation matching regex */
+    const char* const RelationRegex = "^[[:blank:]]*[Rr]elation[[:blank:]]*:" SYMBOL_IDENTIFIER "?$";
```

Comment:
> Does this answer your question?

That wasn't my question at all. But @klokane understood it and addressed my concern.